tests/vcd_role.py

```diff
@@ -26,5 +26,13 @@ def test_get_rights(self):
         rights = role.list_rights()
         assert len(rights) > 0
 
+    def test_create_role(self):
```

Comment:
We can add copyrights header for test files too.

Reply:
fixed.